source/retryable-writes/tests/bulkWrite.yml

```diff
@@ -303,3 +303,63 @@ tests:
                 data:
                     - { _id: 1, x: 12 }
                     - { _id: 2, x: 22 }
+    -
+        description: "Single-document write following deleteMany is retried"
+        failPoint:
+            mode: { times: 1 }
+            data: { failBeforeCommitExceptionCode: 1 }
```

Comment:
@kaloianm confirmed that the `onPrimaryTransactionalWrite` fail point only triggers for write operations that include a transaction ID. Therefore, this will have no effect on the initial `deleteMany` and will trigger once (and only once) for the first `insertOne` attempt. Likewise for the `updateMany` test case below.

Reply:
Cool, could you explain this in a comment in the YML?